chevah/python-package
# [#3355] Update Python distribtuion on Windows for pyopnessl

Scope
=====

We need pyopenssl 16.0.0 to re-enable FTPS tests on Windows.


Changes
=======

Update the versions... as pyopenssl requried an update pycrypto which required an update cffi :)

@alibotean there is an error in the build... it was also present 5 months ago. Can you please check it.

```
./chevahbs: line 179: /c/Users/buildslave/chevah/bs/python-package-win-xp-dev/build/build/python2.7-windows-x86/lib/Lib/site-packages/win32/pythoncomloader27.dll.embedded: No such file or directory
```

I also plan to update the python distro to 2.7.11

How to test
=========

reviewers: @alibotean 

check that the build from testing works for you.

After this is in production we might need to follow-up with a ticket on server to advertise the opnenssl update on windows

thanks!
